.github/workflows/ci-sharktank.yml

```diff
@@ -22,6 +22,43 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  test_punet:
+    name: "Integration Tests - punet"
+    runs-on: nodai-amdgpu-mi250-x86-64
```

Comment:
Would a standard GH runner also suitable for this? See https://github.com/nod-ai/shark-ai/pull/621#discussion_r1861038318.

Reply:
Yeah we can just use `ubuntu-latest`

Reply:
I would rather pin `ubuntu-24.04` and be more specific with the version :) Better to update manually than to see it breaking just because of an update some day.

Reply:
Yes, please use a specific version. `ubuntu-latest` is also going to change to `ubuntu 24` soon: https://github.blog/changelog/2024-11-05-notice-of-breaking-changes-for-github-actions/

Reply:
Changed to `ubuntu-24.04`